patches/server/1055-Implement-get-setResult-for-CartographyInventory.patch

```diff
@@ -0,0 +1,31 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Janet Blackquill <uhhadd@gmail.com>
+Date: Sun, 7 Apr 2024 16:52:19 -0400
+Subject: [PATCH] Implement get/setResult for CartographyInventory
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/inventory/CraftInventoryCartography.java b/src/main/java/org/bukkit/craftbukkit/inventory/CraftInventoryCartography.java
+index af455b3a4eb74339580dc3ec723c832791c555f0..f26cb926103db5b9e6c80aeccbed1249beee1f7c 100644
+--- a/src/main/java/org/bukkit/craftbukkit/inventory/CraftInventoryCartography.java
++++ b/src/main/java/org/bukkit/craftbukkit/inventory/CraftInventoryCartography.java
+@@ -4,8 +4,19 @@ import net.minecraft.world.Container;
+ import org.bukkit.inventory.CartographyInventory;
+ 
+ public class CraftInventoryCartography extends CraftResultInventory implements CartographyInventory {
+-
+     public CraftInventoryCartography(Container inventory, Container resultInventory) {
+         super(inventory, resultInventory);
+     }
++
++    // Paper begin - add getResult/setResult implementation for CartographyInventory
++    @Override
++    public org.bukkit.inventory.ItemStack getResult() {
++        return this.getItem(2);
++    }
++
++    @Override
++    public void setResult(org.bukkit.inventory.ItemStack item) {
++        this.setItem(2, item);
```

Comment:
For the `2` here, use the public static field `CartographyTableMenu.RESULT_SLOT`